lighthouse-core/test/report/html/renderer/psi-test.js

```diff
@@ -59,6 +62,21 @@ describe('DOM', () => {
 
   describe('psi prepareLabData helpers', () => {
     describe('prepareLabData', () => {
+      it('succeeds with LHResult object (roundtrip) input', () => {
+        const roundTripLHResult = /** @type {LH.Result} */ JSON.parse(sampleResultsRoundtripStr);
```

Comment:
this seems good to have, though it makes #6405 a bit more work (e.g. this test will have to trigger a round trip result somehow). Can we prioritize that one soon so all the dependencies on whatever we come up with aren't too ossified?